src/style-spec/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 13.11.0-beta.1
+
+### ✨ Features and improvements
+* Add option to use a feature property as ID for feature state ([#8987](https://github.com/mapbox/mapbox-gl-js/pull/8987))
+* Edit convert functions to output linear interpolation ([#9107](https://github.com/mapbox/mapbox-gl-js/pull/9107)) (h/t [Jo-IE](https://github.com/Jo-IE))
+* Export the internal `derefLayers` utility function from the `style-spec` bundle ([#9121](https://github.com/mapbox/mapbox-gl-js/pull/9121)) (h/t [ahocevar](https://github.com/ahocevar))
+* Update `symbol-avoid-edges` documentation regarding global collision detection ([#9157](https://github.com/mapbox/mapbox-gl-js/pull/9157))
+* Remove reference to deprecated `in` expression ([#9102](https://github.com/mapbox/mapbox-gl-js/pull/9102))
+
+### 🐛 Bug fixes
+* Fix bug where `symbol-sort-key` was not used for collisions that crossed tile boundaries ([#9054](https://github.com/mapbox/mapbox-gl-js/pull/9054))
```

Comment:
This isn't really a style-spec change, right?

Reply:
My thinking was that people who care about the style would care about this because it changes the behavior of `symbol-sort-key`. It's a bug in that the style-spec documentation misspecified the behavior, but now that the behavior matches the documentation it's a mildly breaking change to the semantics of the layout property.